webapp/migrations/Version20230508120033.php

```diff
@@ -19,6 +19,7 @@ public function getDescription(): string
 
     public function up(Schema $schema): void
     {
+        $this->addSql('DELETE from judgetask WHERE submitid not in (SELECT submitid FROM submission)');
```

Comment:
```suggestion
        // Cleanup judgetasks for removed submissions
        // This can happen when a full contest has been removed and the full delete cascade has failed. 
        $this->addSql('DELETE from judgetask WHERE submitid not in (SELECT submitid FROM submission)');
```